types/jasmine/index.d.ts

```diff
@@ -436,58 +483,66 @@ declare namespace jasmine {
 
     interface Env {
         addReporter(reporter: CustomReporter): void;
-
-        execute(): void;
-        describe(description: string, specDefinitions: () => void): Suite;
-        // ddescribe(description: string, specDefinitions: () => void): Suite; Not a part of jasmine. Angular team adds these
-        beforeEach(beforeEachFunction: ImplementationCallback, timeout?: number): void;
-        beforeAll(beforeAllFunction: ImplementationCallback, timeout?: number): void;
-        afterEach(afterEachFunction: ImplementationCallback, timeout?: number): void;
-        afterAll(afterAllFunction: ImplementationCallback, timeout?: number): void;
-        xdescribe(desc: string, specDefinitions: () => void): XSuite;
-        it(description: string, func: () => void): Spec;
+        allowRespy(allow: boolean): void;
+        clearReporters(): void;
+        configuration(): EnvConfiguration;
+        configure(configuration: EnvConfiguration): void;
+        execute(runnablesToRun?: Suite[], onComplete?: Func): void;
+        /**
+         * @deprecated Use hideDisabled option in {@link jasmine.Env.configure} instead.
+         */
+        hideDisabled(value: boolean): void;
+        /**
+         * @deprecated Check hideDisabled option in {@link jasmine.Env.configuration} instead.
+         */
+        hidingDisabled(): boolean;
```

Comment:
This isn't in the docs right now nor was it in the old types. Remove?

Reply:
This is a deprecated method. I figured it was best to include it to make the types more accurate.